server/package.json

```diff
@@ -27,10 +27,12 @@
     "babel-register": "^6.22.0",
     "body-parser": "^1.16.1",
     "dotenv": "^4.0.0",
-    "mongoose": "^4.8.4",
     "nodemon": "^1.11.0"
   },
   "dependencies": {
-    "express": "^4.14.1"
+    "express": "^4.14.1",
+    "mysql": "^2.13.0",
+    "sequelize": "^3.30.4",
+    "sequelize-cli": "^2.7.0"
```

Comment:
I think just the sequelize-cli should be moved, everything else is kind of needed whether in dev or not.